packages/shared/src/components/Settings.tsx

```diff
@@ -164,21 +171,18 @@ export default function Settings({
           >
             Show Weekly Goal widget
           </Switch>
-        </div>
-      </Section>
-      <Section>
-        <SectionTitle>Companion Widget (beta)</SectionTitle>
-        <div className="flex flex-col items-start pl-1.5 -my-0.5">
-          <Switch
-            inputId="hide-companion-switch"
-            name="hide-companion"
-            className="my-3"
-            checked={!optOutCompanion}
-            onToggle={toggleOptOutCompanion}
-            compact={false}
-          >
-            Enable widget on articles
-          </Switch>
+          {companionPlacement !== 'off' && (
+            <Switch
+              inputId="hide-companion-switch"
+              name="hide-companion"
+              className="my-3"
+              checked={!optOutCompanion}
+              onToggle={toggleOptOutCompanion}
```

Comment:
As raised on [slack](https://dailydotdev.slack.com/archives/C03G1FZPTEX/p1653458794644749) we should also request the permission if they are not granted yet here.

Reply:
Oh, okay. Though that would make the behavior of the toggle inconsistent, wouldn't it? Now, it made me question, shouldn't we make this available only if the user has already accepted the permission?

Reply:
Would you mind raising this with Product?

Reply:
Totally cool. Was just finishing the work in API then was about to send a message to the channel. Thank you! 🚢